components/02-form-elements/input/input--password.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="field">
+  {{> @label label_for="inputPassword" label_text="Password"}}
+  <div class="field--toggle">
+    {{> @label label_for="showPasswordToggle" label_text="Show password" label_inline="true" label_class="field__label"}}
+    <input id="showPasswordToggle" class="field__input input input--checkbox" type="checkbox" />
+  </div>
+  <input id="inputPassword" type="password" class="input input--text" aria-describedby="field" />
```

Comment:
Not what the `aria` role id for but it should be an id. Is it supposed to be the "Show password" label? `aria-labelledby` might be a better choice in that case. Worth testing both.